module/validate.go

```diff
@@ -40,6 +40,16 @@ func (m *Module) Execute(targets map[string]pgs.File, pkgs map[string]pgs.Packag
 	for _, f := range targets {
 		m.Push(f.Name().String())
 
+		hasValidate := false
+		for _, file := range f.Imports() {
+			if file.Package().ProtoName().String() == "validate" {
```

Comment:
This is a reasonable heuristic but will have some false positives, like if I had my own `validate` proto that wasn't related to PGV. Can we make this more precise?